build/ci-scripts/install.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/bash
+
+set -x
+
+originalDirectory=$(pwd)
+
+cd ..
+
+mkdir mediawiki
+wget -O- https://github.com/wikimedia/mediawiki/archive/$MW_BRANCH.tar.gz | tar -zxf - -C mediawiki --strip-components 1
```

Comment:
`-f -` is redundant in `tar`, `tar -zx` would work just as well :)

Reply:
> This feels a bit overly complicated to me, to be honest, with things like the `WB` variable that’s only ever set to `"both"` even though we still have code supporting other configurations… not sure how I feel about it.

Well it was a copy and paste job that gave us a lot of flexibility without any real effort. Sure we might not have the need to flip client on or off now but the possibility is there. I guess we could spend more time to trim it down to the bare minimum of what we need right now but that seems ... like a waste of time?

Reply:
> `-f -` is redundant in `tar`, `tar -zx` would work just as well :)

done